### PR TITLE
FAT-4546 Added linkingRuleId field

### PR DESCRIPTION
**Pull request by ArtHov**
[https://issues.folio.org/browse/FAT-4546] added new field linkingRuleId to test

### CODE REVIEW

**Review by hasanepam — APPROVED** [review]

File: mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/samples/links/createTwoLinks.json [hunk 1]
```diff
@@ -10,6 +11,7 @@
       ]
     },
     {
+      "linkingRuleId": 7,
```
Comment (Shans-Kaluhin) [inline mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/samples/links/createTwoLinks.json:14]: Should be strict with linking rules `/linking-rules/instance-authority`
```suggestion
      "linkingRuleId": 1,
```

File: mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/samples/links/createTwoLinks.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "links": [
     {
+      "linkingRuleId": 7,
```
Comment (Shans-Kaluhin) [inline mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/samples/links/createTwoLinks.json:4]: Should be strict with linking rules /linking-rules/instance-authority
```suggestion
      "linkingRuleId": 12,
```

File: mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/samples/links/createLink.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "links": [
     {
+      "linkingRuleId": 7,
```
Comment (Shans-Kaluhin) [inline mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/samples/links/createLink.json:4]: Should be strict with linking rules /linking-rules/instance-authority
```suggestion
      "linkingRuleId": 1,
```

**Comment by Shans-Kaluhin** [comment]
Also please add ticket name to the PR title

File: mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/samples/links/createLink.json (hunk 3 above)
Comment (ArtHov) [reply mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/samples/links/createLink.json]: linkingRuleId changed as in linking rules /linking-rules/instance-authority

**Pull request merged**